submissions/TUstiugov/html-css-popup/index.html

```diff
@@ -0,0 +1,213 @@
+                </ul>
+                <div class="popup__more">
+                  <input type="checkbox" id="more" class="popup__check">
+                  <label for="more" class="popup__label">More</label>
+                  <ul class="popup__more-content popup__container flex-container">
+                    <li>
+                      <a href="#" class="popup__item">
+                        <div class="popup__image">
+                          <img src="./images/popup-icons/chat.png" alt="">
+                        </div>
+                        <p class="popup__subtitle">Chat</p>
+                      </a>
+                    </li>
+
+                    <li>
+                      <a href="#" class="popup__item">
+                        <div class="popup__image">
+                          <img src="./images/popup-icons/classroom.png" alt="">
+                        </div>
+                        <p class="popup__subtitle">Classroom</p>
+                      </a>
+                    </li>
+
+                    <li>
+                      <a href="#" class="popup__item">
+                        <div class="popup__image">
+                          <img src="./images/popup-icons/collection.png" alt="">
+                        </div>
+                        <p class="popup__subtitle">Collection</p>
+                      </a>
+                    </li>
+
+                    <li>
+                      <a href="#" class="popup__item">
+                        <div class="popup__image">
+                          <img src="./images/popup-icons/mars.png" alt="">
+                        </div>
+                        <p class="popup__subtitle">Mars</p>
+                      </a>
+                    </li>
+
+                    <li>
+                      <a href="#" class="popup__item">
+                        <div class="popup__image">
+                          <img src="./images/popup-icons/blogs.png" alt="">
+                        </div>
+                        <p class="popup__subtitle">Blogs</p>
+                      </a>
+                    </li>
+
+                    <li>
+                      <a href="#" class="popup__item">
+                        <div class="popup__image">
+                          <img src="./images/popup-icons/promo.png" alt="">
+                        </div>
+                        <p class="popup__subtitle">Promos</p>
+                      </a>
+                    </li>
+                  </ul>
+                  <p class="popup__footer">Even more from Kottans</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </li>
+        <li class="navigation__item">
+          <a href="#" class="navigation__link navigation__link_bell"></a>
+        </li>
+        <li class="navigation__item">
+          <a href="#" class="navigation__link navigation__link_user"></a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+</body>
+
+</html>
```

Comment:
Put an empty line at the end of every file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1). Tune your code editor's settings so it does this for you.

Reply:
Hi, thanks for review. Done.